### DISCUSSION TITLE
%E5%B0%8E%E8%AE%80%E5%8D%A1/%E7%AC%AC35%E9%80%B1%E5%B0%8E%E8%AE%80%E5%8D%A1

### DISCUSSION THREAD

**Discussion by giscus** [cit-hall-43/cit-hall-43.github.io#6] [Q&A]
# %E5%B0%8E%E8%AE%80%E5%8D%A1/%E7%AC%AC35%E9%80%B1%E5%B0%8E%E8%AE%80%E5%8D%A1

晨興補充材料 出埃及記 第廿八、廿九篇–以色列人過紅海

https://cit-hall-43.github.io/%E5%B0%8E%E8%AE%80%E5%8D%A1/%E7%AC%AC35%E9%80%B1%E5%B0%8E%E8%AE%80%E5%8D%A1.html

**Comment by cit-hall-43** [comment]
『耶穌對他說，狐狸有洞，天空的飛鳥有窩，人子卻沒有枕頭的地方。』路九58
